src/quo2/components/wallet/network_bridge/view.cljs

```diff
@@ -0,0 +1,45 @@
+(ns quo2.components.wallet.network-bridge.view
+  (:require
+    [clojure.string :as string]
+    [quo2.components.icon :as icon]
+    [quo2.components.markdown.text :as text]
+    [quo2.foundations.colors :as colors]
+    [quo2.foundations.resources :as resources]
+    [quo2.theme :as quo.theme]
+    [react-native.core :as rn]
+    [quo2.components.wallet.network-bridge.style :as style]))
+
+(defn network-bridge-internal
+  [{:keys [theme network state amount]}]
```

Comment:
or we can also ask designers to use a different name than `state` :)

Reply:
@J-Son89 I think it depends. If state is unique -- for example it cannot be `disabled` and `locked` at the same time, then there is no need to flatten. We can ask designers about that.

Reply:
Yep good point @OmarBasem - I'm good with either way. 👍

Reply:
I guess it's similar to `type` in other components such as `button`.

Reply:
If `state` becomes a common property in all of pur components, I think we can get used to it.

And yes, we should make sure if states can be mixed (e.g., loading + disabled is possible).

Reply:
I'm okay with the name `state` and `type`. Designers could use different names (e.g. `kind` instead of type to not shadow the core `type` function), but I also like the idea of having a DS that's completely independent of technical details of the programming language in use.

If possible, the less exceptions we make the easier for us to apply this guideline.